### ISSUE TITLE
About learning_rate

### ISSUE THREAD

**Issue by zhongzyd** [eladrich/pixel2style2pixel#114]
In the pape, learning rate is 0.001. Why learning rate is 0.0001 in the code? Which one would you suggest?

**Comment by yuval-alaluf** [comment]
Good catch. Seems to be a typo in the paper. it should be `0.0001`. We'll make the correction in the revision of the paper.
